klippa/tests/integration_test.rs

```diff
@@ -0,0 +1,538 @@
+//! Test subsetter output against expected results.
```

Comment:
I think the name of this file is slightly vague, and I would prefer to call it something like `harfbuzz_tests` or `hb_subset_tests` or something?

Reply:
oh actually this is our own integration test, we'll store our expected files as well